packages/reports/tests/integration/components/filter-builders/base-test.js

```diff
@@ -8,18 +8,22 @@ import Component from '@ember/component';
 import BaseFilterBuilderComponent from 'navi-reports/components/filter-builders/base';
 import $ from 'jquery';
 
-const filter = {
-  subject: {
+const requestFragment = {
+  type: 'dimension',
+  columnMetadata: {
     name: 'Device Type'
-  },
+  }
+};
+
+const filter = {
```

Comment:
Can we simply use this object/shape:

https://github.com/yahoo/navi/blob/request-v2/packages/core/addon/models/bard-request-v2/fragments/filter.ts